### PR TITLE
Added postal_town to switch/case for locality address components

### PR DESCRIPTION
**Pull request by watchinharrison**
The locality for most UK addresses is registered under the type postal_town.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ composer.lock
 composer.phar
 phpunit.xml
 php-cs-fixer.phar
+.idea
```
Comment (willdurand) [inline .gitignore]: would be nice not to change this file :)

**Comment by willdurand** [comment]
Hi! Thanks for contributing. Sounds good to me, however would you mind add a test case?

**Comment by watchinharrison** [comment]
By test case, do you mean a return response from the google geocode api that includes the type postal_town? 

http://maps.googleapis.com/maps/api/geocode/json?address=CF37,%20United%20Kingdom
http://maps.googleapis.com/maps/api/geocode/json?address=SR6,%20United%20Kingdom
http://maps.googleapis.com/maps/api/geocode/json?address=EH33,%20United%20Kingdom

are a few.

**Comment by willdurand** [comment]
@watchinharrison I rather meant this: https://github.com/geocoder-php/Geocoder/commit/47baf0c5a707f9324f4bcbc5e623308160274685#diff-f97d2f35dbaa1150b96b68f1e1869885R348, but I wrote it for you that time ;-)

Thanks you for your patch, it has been added to the `master` branch (cf. 47baf0c5a707f9324f4bcbc5e623308160274685).

**Pull request closed without merging**